consistent capitalization for terms and acronyms per New Relic style (#10945)
newrelic/docs-website: src/content/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.mdx

---

> title: Introduction to single page app (SPA) monitoring

Reviewer: ```suggestion
title: Introduction to single page application (SPA) monitoring
```

tags:
  - Browser
  - Single page app monitoring
  - Get started
translate:
  - jp

> metaDescription: The New Relic Browser SPA monitoring examines your users' experience with the frontend logic for single-page apps or any apps that pull content dynamically.

Reviewer: ```suggestion
metaDescription: New Relic browser SPA monitoring examines your users' experience with the front-end logic for single-page apps or any apps that pull content dynamically.
```

Reviewer: @amyreinholds I hyphenated front-end because it's an adjective to logic.

import browserSpaPageview from 'images/browser_screenshot-full_spa-pageview.png'

> New Relic browser monitoring includes single-page application (SPA) monitoring to give you deeper visibility and actionable insights into real user interactions with single-page apps, and for any app that uses AJAX requests.

Reviewer: ```suggestion
New Relic browser monitoring offers single-page application (SPA) monitoring to give you deeper visibility and actionable insights into real user interactions with single-page apps or any app that uses AJAX requests.
```

Reviewer: @amyreinholds I chose "offers" because not everyone utilizes SPA monitoring, depending on which agent edition they have
